turtlebot2_drivers/CMakeLists.txt

```diff
@@ -12,28 +12,34 @@ find_package(ament_cmake REQUIRED)
 find_package(geometry_msgs REQUIRED)
 find_package(nav_msgs REQUIRED)
 find_package(rclcpp REQUIRED)
-find_package(rmw_implementation_cmake REQUIRED)
-
-include_directories(${kobuki_driver_INCLUDE_DIRS})
-
-macro(targets)
-  if(NOT target_suffix STREQUAL "")
-    get_rclcpp_information("${rmw_implementation}" "rclcpp${target_suffix}")
-  endif()
-  add_executable(kobuki_node${target_suffix} src/kobuki_node.cpp)
-  ament_target_dependencies(kobuki_node${target_suffix}
-    "rclcpp${target_suffix}"
-    "nav_msgs"
-    "geometry_msgs")
-  target_link_libraries(kobuki_node${target_suffix} ${kobuki_driver_LIBRARIES})
-  add_executable(dumb_teleop${target_suffix} src/dumb_teleop.cpp)
-  ament_target_dependencies(dumb_teleop${target_suffix}
-    "rclcpp${target_suffix}"
-    "nav_msgs"
-    "geometry_msgs")
-
-  install(TARGETS kobuki_node${target_suffix} dumb_teleop${target_suffix}
-          DESTINATION ${CMAKE_INSTALL_PREFIX}/bin)
-endmacro()
-
-call_for_each_rmw_implementation(targets GENERATE_DEFAULT)
+find_package(sensor_msgs REQUIRED)
+find_package(tf2_ros REQUIRED)
+
+add_executable(kobuki_node src/kobuki_node.cpp)
+ament_target_dependencies(kobuki_node
+  "geometry_msgs"
+  "nav_msgs"
+  "rclcpp"
+  "sensor_msgs"
+  "tf2_ros")
+# Note that this setting must come after the call to
+# ament_target_dependencies() to avoid including a ROS1 tf2 installation that
+# might be on the system.
+target_include_directories(kobuki_node PRIVATE ${kobuki_driver_INCLUDE_DIRS})
```

Comment:
Yeah, this is an unfortunate reality because the package names are the same. I guess we're getting lucky on the linking? I could see that we would easily use the headers of tf2_ros from ROS 2 but the library of tf2_ros from ROS 1 by accident. Should we consider renaming `tf2_ros` as `tf2_ros2` (as an example) to avoid this issue? We could eventually probably make it so that all the other `tf2*` packages are ABI compatible between ROS 1 and ROS 2, but that will likely never be the case with the `tf2_ros` part.